packages/theme/stories/introduction.stories.mdx

```diff
@@ -121,7 +121,7 @@ Characteristics semantically group both palette and foundational tokens to provi
 | Status                      | Components indicating an informational, warning, error or success state              | `--salt-status-info-borderColor`             | `--salt-palette-info-border`                       |
 | Taggable                    | Components that allow data to be organized                                           | `--salt-taggable-background`                 | `--salt-palette-interact-primary-background`       |
 | Target                      | Components that form a target for draggable elements                                 | `--salt-target-borderStyle`                  | `dashed`                                           |
-| Text                        | All text components or HTML elements                                                 | `--salt-text-primary-foreground`             | `--salt-palette-neutral-primary-foreground`        |
+| Text                        | All text components or HTML elements                                                 | `--salt-foreground-primary`             | `--salt-palette-neutral-primary-foreground`        |
```

Comment:
needs changing to be separate characteristic Foreground
text stays as a different characteristic i assume? havent checked design

Reply:
updated